Keywords/common.resource

```diff
@@ -15,4 +15,4 @@ Open Application
 
 Go to Home Page
     Go To    ${url}
-    Wait Until Element Is Visible    //a[@class="emt_logo"]    timeout=10s
\ No newline at end of file
+    Wait Until Element Is Visible    //a[@class="emt_logo"]    timeout=20s
```

Comment:
Put all the sleep duration in config file

Reply:
since this is not failing any TC can you please accept this PR?
I will work on this feedback in next commit.

Reply:
OK